docs/Users_Guide/scatter.rst

```diff
@@ -0,0 +1,227 @@
+************
+Scatter Plot
+************
+
+Description
+===========
+The scatter plot is useful for illustrating relationships between pairs of continuous variables.
+This plot was developed to support plotting MPR (matched pair) data from the MET point-stat tool.
+**NOTE:** This MET output data must first be reformatted into a format that can be read in by the scatter plot code.
```

Comment:
Suggest using the note directive (.. note:) to be consistent with other notes in the documentation.

Reply:
done